composer.json

```diff
@@ -277,7 +277,10 @@
             "phpunit"
         ],
         "check-cs": "vendor/bin/ecs check --ansi",
-        "fix-cs": "vendor/bin/ecs check --fix --ansi",
+        "fix-cs": [
+            "vendor/bin/ecs check --fix --ansi",
+            "vendor/bin/ecs check-markdown README.md"
```

Comment:
I've just noticed.
Does this fix or check? The `--fix` behaviour should be the same as in `check` command

Reply:
It is for fix. Ok, I will try add `--fix` as well for `check-markdown` and `check-heredoc-nowdoc`

Reply:
added `--fix` option, need new tag of `easy-coding-standard` first

Reply:
Tagged :+1: